mon-pix/translations/fr.json

```
@@ -1686,6 +1686,15 @@
           "title": "Transcription"
         }
       },
+      "navigation": {
+        "buttons": {
+          "aria-label": {
+            "disabled": "Non disponible. Terminer l'étape précédente",
+            "enabled": "Accéder à {sectionTitle}",
```

En faisant des tests avec voiceover, je me dis que l'utilisateur peut ne pas s'avoir ce qu'est par Exemple S'exercer. Peut être ajouter la notion de section ?
```suggestion
            "enabled": "Accéder à la section {sectionTitle}",
```

---

On se disait (avec Diane et Matthias) que l'utilisateur n'a pas connaissance du terme "section", on était donc plus parti sur le mot "étape" (mot présent dans le bouton disabled du coup)
T'en penses quoi ?

---

oui le terme "étape" me convient aussi 👍

---

revu ensemble : c'est bien étape qu'il faut utiliser 😆